oeps/oep-0014.rst

```diff
@@ -0,0 +1,175 @@
+| Author        | Christina Roberts, christina@edx.org      |
++---------------+-------------------------------------------+
+| Arbiter       | TBD                                       |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Process                                   |
++---------------+-------------------------------------------+
+| Created       | 2017-01-09                                |
++---------------+-------------------------------------------+
+| `References`  | `ORA PR Discussion`_,                     |
+|               | `Initial Archiving Discussions`_          |
++---------------+-------------------------------------------+
+
+.. _ORA PR Discussion: https://github.com/edx/edx-ora/pull/187
+.. _Initial Archiving Discussions: https://openedx.atlassian.net/wiki/display/IT/Proposed+Github+Deprecation+Process
+
+Abstract
+========
+
+The `edx organization`_ contains a large number of repositories, most of
+which are active and maintained, but some of which are now obsolete. To
+clarify the status of repositories, and to ensure that we are only monitoring
+active repositories via Gemnasium (which alerts about the usage of third
+party libraries with known security issues), a process for archiving a
+repository is defined below.
+
+.. _edx organization: https://github.com/edx
+
+Motivation
+==========
+
+Recently openedx.yaml files were added to edX repositories per
+`OEP-2 <oep-0002.rst>`_. In the course of deciding owners for those repositories,
+there was a `ORA PR Discussion`_ about how best to handle deprecated or obsolete
+repositories; in particular, do obsolete repositories need owners, and how can
+repositories be clearly marked as present for archive purposes only?
+
+This discussion resurfaced related to edX's usage of Gemnasium to report
+the usage of third-party libraries that have known security issues. All
+repositories under the edX organization were being monitored, but this added
+noise when trying to understand the number of third-party library updates
+required for actively maintained repositories.
+
+Specification
+=============
+
+When a repository under the `edx organization`_ will no longer be maintained
+because it is no longer in use, the following steps should be followed:
+
+First, send an email to edx-code@googlegroups.com announcing that the
+repository will be archived, and inquiring if anyone would like to take
+ownership of the repo. If someone wishes to take ownership, transfer the
+repository to their github organization and hand over rights [Open questions:
+Do we need to outline these steps further? Would there need to be internal
+sign-off before transferring the repo outside of edX? Would we wish to create
+our own fork of the repo?].
+
+If nobody wishes to take ownership of the repository then:
```

Comment:
@cahrens Regarding:

> The main problem I'm having is that I don't know how we actually come to a decision, as feelings are pretty split among the 3 different proposals.

Short story: My understanding is that at this point, you decide. 

Long story: Broadly speaking, the goal of the process is to give a chance to solicit feedback from as many stakeholders as possible. Having and understanding broad input from diverse viewpoints leads to better design decisions. That doesn't mean there needs to be consensus -- just an understanding of the viewpoints and trade-offs by the author of the PR. Ideally, everyone has a chance to kibitz, but ultimately, the person driving the thing gets to decide. The only people with blocking power are two reviewers (and, at this point, you have three :+1:'s), your lead architect, and the chief architect. 

100% consensus is rarely good design. Some people will not be fully familiar with the problem. In some cases, compromises between people with conflicting styles and design practices lead to worse overall solutions than one consistent with one. Requiring 100% consensus also tends to lead to people soliciting less feedback, which almost always leads to worse design.